### PR TITLE
chore(pie-webc-testing): DSW-000 bump playwright

### PR DESCRIPTION
**Pull request by sand4rt**
## Describe your changes (can list changeset entries if preferable)

Was checking out your library and saw playwright could use an upgrade

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: packages/components/pie-modal/playwright-lit-visual.config.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import { defineConfig } from '@sand4rt/experimental-ct-web';
-import { getPlaywrightVisualConfig } from '@justeattakeaway/pie-components-config';
+import { defineVisualConfig } from '@justeattakeaway/pie-components-config';
```
Comment (xander-marjoram) [inline packages/components/pie-modal/playwright-lit-visual.config.ts:1]: Thanks for the PR! Should this be `definePlaywrightVisualConfig`?

Comment (sand4rt) [reply packages/components/pie-modal/playwright-lit-visual.config.ts]: fixed

**Comment by siggerzz** [comment]
I have a slight feeling these CI checks will fail given you're working out of a fork, and our CI relies on some of our tokens, but we'll see 👀

**Review by ashleynolan — APPROVED** [review]
Thanks for the contribution – this is really useful 👍

**Comment by xander-marjoram** [comment]
The failing check does not seem related to your PR. Let's see if it fails again after another PR is merged.

**Comment by ashleynolan** [comment]
The lint issues are linked to the new webc icon package. This won't be an issue when @xander-marjoram's refactor PR for the icons is merged, so shouldn't stop this PR going in.

**Pull request merged**